docs/source-pytorch/accelerators/hpu_troubleshoot.rst

```diff
@@ -0,0 +1,38 @@
+:orphan:
+
+.. _hpu_troubleshoot:
+
+Accelerator: HPU training
+=========================
+**Audience:** Gaudi chip users looking to troubleshoot their model changes during training with HPU.
+
+----
+
+Runtime Errors
+--------------
+
+This section provides troubleshooting instructions that can be referred to for common issues when training PyTorch models. The following are common functional issues that may occur when running on HPU and not on CPU/GPU.
+
+The following table outlines some possible runtime errors:
```

Comment:
@jerome-habana Thanks for starting a troubleshooting doc.

The Table with solutions doesn't seem specific to the HPU Accelerator, as it is generic to all accelerators.

Reply:
I agree @kaushikb11, we need a mechanism for users who search with the error codes to get resolution. As of now, I dont see any documentation that aids with it